tools/depends/target/ffmpeg/FFMPEG-VERSION

```diff
@@ -1,5 +1,6 @@
 LIBNAME=ffmpeg
 BASE_URL=https://github.com/xbmc/FFmpeg/archive
-VERSION=2.8.6-Jarvis-16.1
+BRANCH=release/2.8-xbmc
```

Comment:
having a "/" in the name does not work. needs to be something like 2.8-kodi

@wsnipex can we pull the sha from HEAD and trigger a re-build? currently depends just compares the names, right?